### PR TITLE
Add FastChat accuracy test script

### PR DESCRIPTION
**Pull request by anw90**
Usage:
```
cd ./benchmarks/accuracy
bash run.sh
```

### CODE REVIEW

File: benchmarks/accuracy/README.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+You can simply execute the `run.sh` script to perform all the steps.
+
+## Main Files
+
+* run.sh
+
+    The script integrates all the steps.
+
+    ```bash
+    bash ./run.sh <local_model_dir>
```
Comment (Seventeen17) [inline benchmarks/accuracy/README.md]: What's difference between `ORIGINAL_MODEL_DIR` and `local_model_dir`?

File: benchmarks/accuracy/README.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+The Accuracy Benchmark evaluates the performance of TorchAcc using [FastChat](https://github.com/AlibabaPAI/FastChat_TorchAcc) against a baseline established by Torch native. The benchmark aims to ensure that TorchAcc maintains comparable accuracy levels with Torch native.
+
+## Evaluation Process
+
+To perform the evaluation, follow these steps:
+
+1. Set Baseline
+
+    ```bash
+    bash ./llama.sh <ORIGINAL_MODEL_DIR> 0
```
Comment (Seventeen17) [inline benchmarks/accuracy/README.md:14]: `0, 1` means use torchacc or not?

**Review by baoleai — APPROVED** [review]

File: benchmarks/accuracy/README.md (hunk 2 above)
Comment (anw90) [reply benchmarks/accuracy/README.md:14]: yeah.

```
if [[ $# -ne 2 && $# -ne 3 ]]; then
  echo "Usage: $0 <local_model_dir> <use_torchacc> [checkpiont_output_dir]"
  echo "  local_model_dir: Path to the local directory where the model will be saved."
  echo "  use_torchacc: 0 or 1 to indicate whether to use TorchAcc."
  echo "  checkpoint_output_dir: Optional. Default is the model name in <local_model_dir>."
  exit 1
fi
```

File: benchmarks/accuracy/README.md (hunk 1 above)
Comment (anw90) [reply benchmarks/accuracy/README.md]: In the Evaluation Process, ORIGINAL_MODEL_DIR refers to the checkpoint downloaded from HuggingFace or ModelScope. This name is used to differentiate between TORCH_NATIVE_CHECKPOINT and TORCHACC_CHECKPOINT, making the evaluation process more clearly. The local_model_dir mentioned here allows you to specify any checkpoint path. This explanation clarifies the usage of the bash script.

File: benchmarks/accuracy/README.md (hunk 2 above)
Comment (Seventeen17) [reply benchmarks/accuracy/README.md:14]: I've noticed this code, but users may be confused when reading the README, perhaps it should be explained in the README?

File: benchmarks/accuracy/README.md (hunk 1 above)
Comment (Seventeen17) [reply benchmarks/accuracy/README.md]: Maybe I missed it, it seems that when users don't specify checkpoint_output_dir but provides the correct number of parameters, the Bash log won't output where the checkpoint is written to. We need to get the source code to know parameters' means for sure?

Comment (anw90) [reply benchmarks/accuracy/README.md]: Using run.sh, users do not need to know the output checkpoint path. I will add some explanation in the documentation.

File: benchmarks/accuracy/README.md (hunk 2 above)
Comment (anw90) [reply benchmarks/accuracy/README.md:14]: Ok, will do.

**Review by Seventeen17 — APPROVED** [review]

File: benchmarks/accuracy/README.md (hunk 1 above)
Comment (anw90) [reply benchmarks/accuracy/README.md]: done

File: benchmarks/accuracy/README.md (hunk 2 above)
Comment (anw90) [reply benchmarks/accuracy/README.md:14]: done

**Pull request merged**